internal/impl/kube.go

```diff
@@ -190,6 +161,91 @@ type KubeConfig struct {
 	//
 	// [1] https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#example-MustParse.
 	Resources resourceRequirements
+
+	// Probes to check the readiness/liveness of the pods.
+	Probe map[string]ProbeOptions
```

Comment:
What do you think about having a field for each probe?

```go
LivenessProbe ProbeOptions
ReadinessProbe ProbeOptions
```

Reply:
Done.